vassal-app/src/main/java/VASSAL/counters/FreeRotator.java

```diff
@@ -203,6 +203,8 @@ public double getAngle() {
     return useUnrotatedShape ? 0.0 : validAngles[angleIndex];
   }
 
+  // These are deprecated and also don't work because MatCargo also rotates things
+  @Deprecated
```

Comment:
Should tack a `(since = "2021-11-20", forRemoval = true)` onto these so they get picked up by the deprecation checker. Since these are broken, it would be good to encourage people not to use them, even if we don't ever actually remove them.

Reply:
Added the for-removals. I don't really have a test case because this was just a drive-by-noticing-the-code rather than someone sent me a broken module.